streamlit.runtime.media_file_storage.MediaFileStorageError: Error opening 'assets/alert-icon.svg'
Traceback:
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/streamlit/runtime/scriptrunner/exec_code.py", line 88, in exec_func_with_error_handling
    result = func()
             ^^^^^^
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 579, in code_to_exec
    exec(code, module.__dict__)
File "/home/runner/workspace/main.py", line 26, in <module>
    st.image("assets/alert-icon.svg")
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/streamlit/runtime/metrics_util.py", line 409, in wrapped_func
    result = non_optional_func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/streamlit/elements/image.py", line 172, in image
    marshall_images(
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/streamlit/elements/lib/image_utils.py", line 438, in marshall_images
    proto_img.url = image_to_url(
                    ^^^^^^^^^^^^^
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/streamlit/elements/lib/image_utils.py", line 297, in image_to_url
    url = runtime.get_instance().media_file_mgr.add(image, mimetype, image_id)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/streamlit/runtime/media_file_manager.py", line 226, in add
    file_id = self._storage.load_and_get_id(
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/streamlit/runtime/memory_media_file_storage.py", line 115, in load_and_get_id
    file_data = self._read_file(path_or_data)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/streamlit/runtime/memory_media_file_storage.py", line 167, in _read_file
    raise MediaFileStorageError(f"Error opening '{filename}'") from ex